modules/apps/jaggery/emm/src/emm/units/role-listing/public/js/role-listing.js

```diff
@@ -156,7 +156,7 @@ $("#role-grid").on("click", ".remove-role-link", function () {
 
 $("#search-btn").click(function () {
     var searchQuery = $("#search-by-name").val();
-    if (!searchQuery.trim() == "") {
+    if (searchQuery.trim() != "") {
```

Comment:
I think there's something wrong with the logic. It seems like this should change to -
var searchQuery = $("#search-by-name").val(); 
searchQuery = searchQuery.trim();
if (searchQuery) {